lesson-4-metrics-logs-observability/README.md

```diff
@@ -0,0 +1,98 @@
+Microservices Perf\&Scale testing
+=================================
+
+What is handy to know:
+
+* What is percentile (and histogram and average)
+* Sequential vs. parallel
+* Throughput vs. latency
+* Horizontal vs. vertical scaling
+* Understand pod resource **requests** and **limits** (quality of service for testng vs. prod settings)
+
+![Percentiles, Percentile Rank & Percentile Range: Definition & Examples](/assets/percentile.png "Percentiles, Percentile Rank & Percentile Range")
```

Comment:
please use the original SVGs, there shall be the least amount of binary blobs in git

Reply:
Done. Please take a look now.